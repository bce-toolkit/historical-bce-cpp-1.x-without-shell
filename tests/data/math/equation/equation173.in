5 3
1 0 -2 -4 0
3 0 -8 0 0
0 1 -1 -4 0
# NH3+S=(NH4)2S+N4S4
