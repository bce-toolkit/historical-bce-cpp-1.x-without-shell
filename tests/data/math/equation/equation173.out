16Xa
10Xa
6Xa
Xa
